test/controllers/users_controller_test.rb

```diff
@@ -4,6 +4,7 @@ class UsersControllerTest < ActionController::TestCase
 
   def setup
     @request.env["devise.mapping"] = Devise.mappings[:user]
+    Game.instance.reload_user_count
```

Comment:
Should be possible to remove this if `users_count` is properly initialized in the teams fixture.